interfaces/builtin/basedeclaration.go

```diff
@@ -138,6 +138,9 @@ authority-id: canonical
 series: 16
 revision: 0
 plugs:
+  classic-support:
+    allow-installation: false
+    deny-auto-connection: true
```

Comment:
@jdstrand Do we actually need this? Won't the slot already prevent both the installation and the auto-connection unless it's the core snap?

Same question for docker-support below.